app/views/outcomes-report/PageOutcomesReport.vue

```diff
@@ -191,53 +201,66 @@ export default {
-        if (this.includeOther) {
-          await this.fetchUsingBackgroundJob({ kind, orgIdOrSlug, includeSubOrgs, country, startDate, endDate, includeOther: true })
-        }
+        stats = await this.fetchUsingBackgroundJob({ kind, orgIdOrSlug, includeSubOrgs, country, startDate, endDate, includeOther: this.includeOther })
       }
       this.setStats({ stats, includeSubOrgs, kind })
     },
 
     setStats ({ stats, includeSubOrgs, kind }) {
       if (!stats) return
+      this.org = this.setOrgAndSubOrgs(stats)
+      if (this.includeOther) {
+        this.otherOrg = this.setOrgAndSubOrgs(stats, 'other')
+      }
+    },
+
+    setOrgAndSubOrgs (stats, product = 'current') {
       let subOrgs = []
-      if (includeSubOrgs) {
-        for (const childKind of orgKinds[kind].childKinds) {
-          subOrgs = subOrgs.concat(stats.current[childKind + 's'] || [])
+      if (this.includeSubOrgs) {
+        for (const childKind of orgKinds[this.kind].childKinds) {
+          subOrgs = subOrgs.concat(stats[product][childKind + 's'] || [])
         }
         for (const [index, subOrg] of subOrgs.entries()) {
           subOrg.initiallyIncluded = Boolean(!subOrg.archived && index < this.subOrgLimit && subOrg.progress && subOrg.progress.programs && (subOrgs.length > 1 || this.subOrgLimit === 1))
           // TODO: better way to get rid of redundant info if there is only one subOrg
         }
       }
-      this.subOrgs = Object.freeze(subOrgs) // Don't add reactivity
+      if (product === 'current') {
+        this.subOrgs = Object.freeze(subOrgs) // Don't add reactivity
+      } else {
+        this.otherSubOrgs = Object.freeze(subOrgs) // Don't add reactivity
+      }
 
-      const licenses = stats.current.licenses
-      const orgs = stats.current[kind + 's']
+      const licenses = stats[product].licenses
+      const orgs = stats[product][this.kind + 's']
       if (orgs) {
         orgs[0].subOrgs = this.subOrgs
         orgs[0].newLicenses = licenses
-        this.org = Object.freeze(orgs[0]) // Don't add reactivity
-        console.log('   ... got our org', this.org)
+        const org = Object.freeze(orgs[0]) // Don't add reactivity
+        console.log('   ... got our org', org)
+        return org
       }
     },
 
     async fetchUsingBackgroundJob ({ kind, orgIdOrSlug, includeSubOrgs, country, startDate, endDate, includeOther }) {
-      const otherProduct = includeOther ? this.otherProduct : undefined
-      const resp = await createJob(JOB_TYPE, { kind, orgIdOrSlug, includeSubOrgs, country, dateRange: { startDate, endDate } }, otherProduct)
+      const resp = await createJob(JOB_TYPE, { kind, orgIdOrSlug, includeSubOrgs, country, dateRange: { startDate, endDate } })
+      let resp2
+      if (includeOther) {
+        resp2 = await createJob(JOB_TYPE, { kind, orgIdOrSlug, includeSubOrgs, country, dateRange: { startDate, endDate } }, this.otherProduct)
```

Comment:
there is option `callOz` which can be used to call ozaria equivalent URL, you can pass that option instead from here or createJob function

Reply:
oh okay. but we need to update that `callOz` option to `callOther` to support callCoCo too. so I'll just add a todo here